### DISCUSSION TITLE
Kitty graphics protocol is not working in tmux

### DISCUSSION THREAD

**Discussion by gazzaar** [ghostty-org/ghostty#3712] [Q&A] [answered]
The image preview works fine in Ghostty, but it does not work when I open a tmux session.
- I use  macOS Sequoia 15.2 arm64
- I use yazi as my file manager

- Ghostty session 
![ghostty](https://github.com/user-attachments/assets/2b768481-9a8f-40d3-8af2-03678bfcb18b)

- Tmux session
![tmux](https://github.com/user-attachments/assets/26805715-e9f2-48c1-87fb-e20cecf1ae74)

**Comment by gazzaar** [comment]
- I think it's not related to tmux because it works just fine here in wezterm: 

![SCR-20241228-mveo](https://github.com/user-attachments/assets/83935265-f4b1-4bab-9abe-b181159160f6)

**Comment by 00-kat** [comment]
Works perfectly fine with zero tmux config for me.

**Reply by gazzaar** [reply]
I'm not sure why it's not working for me, maybe I'll reinstall tmux again to see, or kill the server and all sessions to start a fresh one.

**Reply by ghost** [reply]
> I'm not sure why it's not working for me, maybe I'll reinstall tmux again to see, or kill the server and all sessions to start a fresh one.

hey, did you add this to tmux config : 

```
set -g allow-passthrough on
```

Otherwise image cannot be seen. (According to yazi docs)

**Reply by gazzaar** [reply, marked as answer]
yes I have this option on my config.
It works now guys when I **killed the server and opened a new one**. Thanks!
